src/sota_tools/CMakeLists.txt

```diff
@@ -142,15 +142,10 @@ if (BUILD_SOTA_TOOLS)
         COMMAND ${PROJECT_SOURCE_DIR}/tests/sota_tools/test-bad-option $<TARGET_FILE:garage-push>
         WORKING_DIRECTORY ${CMAKE_CURRENT_SOURCE_DIR})
 
-    # Detect that the thing we passed in to --repo is not a repository
-    add_test(NAME invalid-repo
-        COMMAND ${PROJECT_SOURCE_DIR}/tests/sota_tools/test-invalid-repo $<TARGET_FILE:garage-push>
-        WORKING_DIRECTORY ${CMAKE_CURRENT_SOURCE_DIR})
-
-    # Detect a --repo that isn't in 'archive-z2' mode
-    add_test(NAME bare-mode-repo
-        COMMAND ${PROJECT_SOURCE_DIR}/tests/sota_tools/test-bare-mode-repo $<TARGET_FILE:garage-push>
-        WORKING_DIRECTORY ${PROJECT_SOURCE_DIR}/tests/)
```

Comment:
I definitely support moving some of these scripts into proper Google Test cases. But if these old test scripts are no longer necessary, you should also delete them.